tests/test_filter_detections_stage.py

```diff
@@ -14,20 +14,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
 from unittest import mock
 
 import cupy as cp
 import pytest
 
-from morpheus._lib.common import FileTypes
 from morpheus._lib.common import FilterSource
-from morpheus.io.deserializers import read_file_to_df
 from morpheus.messages import MultiResponseMessage
 from morpheus.messages import ResponseMemory
 from morpheus.messages.message_meta import MessageMeta
 from morpheus.stages.postprocess.filter_detections_stage import FilterDetectionsStage
-from utils import TEST_DIRS
+
+
+@pytest.fixture
+def df_type():
+    """
+    The tests in this module all require a cudf dataframe
+    """
+    yield 'cudf'
```

Comment:
Mark tests with `use_cudf` if they should only be run with cuDF (I just added this feature)